apps/docs/src/basics/breakpoints-grid.md

```diff
@@ -2,7 +2,7 @@
 
 Column grids in ui design and development establish a structured layout, organizing content, components and elements cohesively. They provide a framework for alignment and distribution, ensuring consistency and hierarchy. Additionally, grids serve as the foundation for smooth responsive behavior, allowing seamless adoption to different screen sizes and resolutions, ultimately enhancing usability and overall user experience.
 
-To see the grid and breakpoints in action, feel free to have a look on the [grid playground](https://storybook.onyx.schwarz/iframe.html?args=&id=examples-gridplayground--default).
+To see the grid and breakpoints in action, feel free to have a look on the [grid playground](https://storybook.onyx.schwarz/?path=/story/examples-gridplayground--default).
```

Comment:
didn't we talk about linking the popped-out mode
```suggestion
To see the grid and breakpoints in action, feel free to have a look on the [grid playground](https://storybook.onyx.schwarz/iframe.html?id=examples-gridplayground--default&viewMode=story).
```
in the vitepress docs in the refinement?

Reply:
I though it is more tidely connected to our Storybook if we use the regular link instead of the iframe :)
What do you think? We now have the info text with the full screen mode